test/integration/bench/test_bench.py

```diff
@@ -0,0 +1,29 @@
+import os
+import signal
+from subprocess import PIPE, STDOUT, Popen
+from threading import Thread
+
+from utils import consume_lines, run_command_and_get_output
+
+
+def test_bench(cartridge_cmd, request, tmpdir):
+    base_cmd = [
+        cartridge_cmd, 'bench', '--duration=1'
+    ]
+    tarantool_cmd = ["tarantool",
```

Comment:
https://www.python.org/dev/peps/pep-0008/

I prefer the option as you have decorated above (`base_cmd`)